usaspending_api/conftest.py

```diff
@@ -191,8 +192,16 @@ def mock_matviews_qs(monkeypatch):
 
 
 def pytest_configure():
-    # this function used to remove all non-default DB connections. We undid this 10/2019 to allow testing of loaders
-    pass
+    # This function re-added so tests would stop running against non-test databases.
+    #
+    # To make sure the test setup process doesn't try
+    # to set up another test db, remove everything but the default
+    # DATABASE_URL from the list of databases in django settings
+    test_db = settings.DATABASES.pop(DEFAULT_DB_ALIAS, None)
+    settings.DATABASES.clear()
+    settings.DATABASES[DEFAULT_DB_ALIAS] = test_db
+    # Also remove any database routers
+    settings.DATABASE_ROUTERS.clear()
```

Comment:
```    
    # To make sure the test setup process doesn't try
    # to set up another test db, remove everything but the default
    # DATABASE_URL from the list of databases in django settings
    test_usaspending_db = settings.DATABASES.pop(DEFAULT_DB_ALIAS, None)
    test_broker_db = settings.DATABASES.pop("data_broker", None)
    settings.DATABASES.clear()
    settings.DATABASES[DEFAULT_DB_ALIAS] = test_usaspending_db
    settings.DATABASES["data_broker"] = test_broker_db
    # Also remove any database routers
    settings.DATABASE_ROUTERS.clear()```

Reply:
This should use the `data_broker` database in `settings.DATABASES` that is defined in travis.yml